configure.py

```diff
@@ -560,9 +560,12 @@ def import_string(import_name, silent=False):
             modname = module + '.' + obj
             __import__(modname)
             return sys.modules[modname]
-    except ImportError, e:
+    except ImportError as e:
         if not silent:
-            raise ImportStringError(import_name, e), None, sys.exc_info()[2]
+            if sys.version_info.major == 2:
+                exec("raise ImportStringError(import_name, e), None, sys.exc_info()[2]")  # Get around SyntaxError in Py3
```

Comment:
Came here to submit exactly the same patch. Minor note here -- you could replace this whole `if/else` block with
```
six.reraise(ImportStringError, ImportStringError(import_name, e), sys.exc_info()[2])
```

assuming you've imported six and added it to the dependencies in setup.py